Competition2017/src/competition/subsystems/shoot_fuel/StopFeedingAndCollectionCommandGroup.java

```diff
@@ -0,0 +1,21 @@
+package competition.subsystems.shoot_fuel;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.agitator.AgitatorsManagerSubsystem;
+import competition.subsystems.collector.commands.StopCollectorCommand;
+import competition.subsystems.shooter_belt.ShooterBeltsManagerSubsystem;
+import competition.subsystems.shooter_wheel.ShooterWheelsManagerSubsystem;
+
+public class StopFeedingAndCollectionCommandGroup extends StopAllShootingCommandGroup {
```

Comment:
Yeah, this felt particularly wrong while implementing it but I think it's the best structure that we can get right now. I needed to preserve the injectability of the "shoot and cross baseline" command, so this is what we end up with :grin: